transfers/transfer.py

```diff
@@ -165,6 +166,15 @@ def get_status(am_url, am_user, am_api_key, unit_uuid, unit_type, session, hide_
             response = requests.delete(url, params=params)
             LOGGER.debug('Response: %s', response)
 
+        # If complete, delete source files
+        if delete_on_complete and unit_info and unit_info['status'] == 'COMPLETE':
+            LOGGER.info('Deleting source files for SIP %s from watched directory', db_unit.uuid)
+            try:
+                shutil.rmtree(db_unit.path)
+                LOGGER.info('Source files deleted for SIP %s deleted', db_unit.uuid)
+            except OSError as e:
+                LOGGER.info('Error deleting source files: %s', e)
```

Comment:
@timothyryanwalsh should this be a `warning` instead of `info`?

Reply:
Yes, good catch! I've made the change.